README.md

```diff
@@ -2,7 +2,7 @@
 
 This repository contains a complete example that grabs device data from IoT-Network server, stores it in a database, and then displays the data using a web-based dashboard.
 
-You can set this up on a Docker droplet from [Digital Ocean](https://www.digitalocean.com/) (or on a Ubuntu VM from [DreamCompute](https://www.dreamhost.com/cloud/computing/), or on a "Ubuntu + Docker" VM from the [Microsoft Azure store](https://portal.azure.com/) ) with minimal effort. You should set up this service to run all the time so as to capture the data from your devices; you then access the data at your convenience using a web browser.
+You can set this up on a Docker droplet from [Digital Ocean](https://www.digitalocean.com/) (or on an Ubuntu VM from [DreamCompute](https://www.dreamhost.com/cloud/computing/), or on an "Ubuntu + Docker" VM from the [Microsoft Azure store](https://portal.azure.com/)) with minimal effort. You should set up this service to run all the time to capture the data from your devices; you then access the data at your convenience using a web browser.
```

Comment:
US usage: "a Ubuntu" (because it's generally pronounced "a Youbuntu" in the US, and therefore there's an implied consonant ("Y") that causes "an" to become "a".

Reply:
It's done.